src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -40,10 +40,11 @@ check_device_available() {
 
         case "$DEVICE" in
             chrome)
-                [ -x "$(command -v google-chrome)" ] || [ -x "$(command -v google-chrome-stable)" ] ] || failed=true
+                [ -x "$(command -v chrome)" ] || [ -x "$(command -v google-chrome)" ] ||
```

Comment:
next time, could move the `-x ...` commands to their own lines for better readablity.